### PR TITLE
Add 'Copy Model' context menu option

### PR DESCRIPTION
**Pull request by adamnejm**
Made it play a sound, because I'm cool like that

### CODE REVIEW

File: lua/sh-tas-utils/copymodelproperty.lua [hunk 1]
```diff
@@ -0,0 +1,18 @@
+
+AddCSLuaFile()
```
Comment (Derpius) [inline lua/sh-tas-utils/copymodelproperty.lua:2]: nit: We're doing this automatically for `sh-tas-utils` and `cl-tas-utils` here: https://github.com/TAServers/gmod-utils/blob/master/lua/autorun/tas_utils_init.lua#L13-L16

File: lua/sh-tas-utils/copymodelproperty.lua [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+properties.Add("copy_model", {
+	MenuLabel = "Copy Model",
+	Order = 2137,
```
Comment (Derpius) [inline lua/sh-tas-utils/copymodelproperty.lua:6]: Seems like an oddly specific number, but looking at the docs it doesn't matter much anyway

File: lua/sh-tas-utils/copymodelproperty.lua [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+	MenuIcon = "icon16/page_copy.png",
+	Filter = function(self, ent, ply)
+		if not ent or not ent:IsValid() or ent:IsPlayer() then return end
+		return true
```
Comment (Derpius) [inline lua/sh-tas-utils/copymodelproperty.lua:10]: nit: Can simplify this down to returning the condition
```suggestion
		return ent and ent:IsValid() and not ent:IsPlayer()
```

**Review by Derpius — APPROVED** [review]
Couple of nits, but otherwise looks good!

File: lua/sh-tas-utils/copymodelproperty.lua (hunk 2 above)
Comment (adamnejm) [reply lua/sh-tas-utils/copymodelproperty.lua:6]: Meme number.

**Comment by Derpius** [comment]
# QA Result: Passed ✅ 

- Copy model item is available in the context menu for valid ents with the correct icon ✅ 
- Clicking copies the model path to my clipboard, and plays a sound ✅ 
- Copy model item is not available for invalid ents or players ⚠️ 
  - I was unable to test this as players don't seem to be right-clickable in the context menu.

**Pull request merged**